Flow.Launcher.Core/Resource/Internationalization.cs

```diff
@@ -40,7 +39,7 @@ private void AddFlowLauncherLanguageDirectory()
         }
 
 
-        private void AddPluginLanguageDirectories()
+        internal void AddPluginLanguageDirectories()
```

Comment:
what do you think about making this method accepting a `IEnumerable` or something similar instead of referencing back to the PluginManager?

Reply:
And pass it from PluginManager like this? Seems logical to me.
```csharp
InternationalizationManager.Instance.AddPluginLanguageDirectories(GetPluginsForInterface<IPluginI18n>());
```